public/app/AppWrapper.tsx

```diff
@@ -123,3 +146,11 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
     );
   }
 }
+
+const getStyles = (navPosition: typeof contextSrv.user.navPosition) => ({
+  grafanaApp: css`
+    @media only screen and (min-width: 769px) {
```

Comment:
not sure how i can access the theme here, hence the hardcoded value instead of using the theme breakpoint 🤔

Reply:
You can extend the Props interface with Themable2 and use withTheme2 I think